build.gradle

```diff
@@ -13,11 +13,11 @@ configurations {
 }
 
 group 'org.jboss.aerogear'
-version '2.5.4-SNAPSHOT'
+version '3.0.0-SNAPSHOT'
 
 apply plugin: 'java'
 
-sourceCompatibility = 1.8
+sourceCompatibility = 17
```

Comment:
This makes it incompatible with Keycloak <21

Reply:
I think the KC images before KC 21 use Java 11. Using sourceLevel 11 should fix that.

Reply:
thanks, that works for KC 21 and older versions too!